### PR TITLE
Support Koto

### PR DESCRIPTION
**Pull request by Harumaki-cryptocoin**
Add [Koto](https://ko-to.org)'s support.
\*Passed Test

### CODE REVIEW

**Comment by knocte** [comment]
Should be tabs, not spaces.

**Comment by Harumaki-cryptocoin** [comment]
Thank you for your response!  
I replaced space to tab.

File: NBitcoin.Altcoins/Koto.cs [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+{
+	// Reference: https://github.com/KotoDevelopers/koto/blob/master/src/chainparams.cpp
+	public class Koto : NetworkSetBase
+	{
+	public static Koto Instance { get; } = new Koto();
+	public override string CryptoCode => "KOTO";
+	private Koto()
+	{
+		
+	}
```
Comment (knocte) [inline NBitcoin.Altcoins/Koto.cs]: wrong indentation in the code above ^

File: NBitcoin.Altcoins/AltcoinNetworkSets.cs [hunk 2]
```diff
@@ -27,6 +27,7 @@ public class AltNetworkSets
 		public static Colossus Colossus { get; } = Colossus.Instance;
 		public static GoByte GoByte { get; } = GoByte.Instance;
 		public static Liquid Liquid { get; } = Liquid.Instance;
+	        public static Koto Koto { get; } = Koto.Instance;
```
Comment (knocte) [inline NBitcoin.Altcoins/AltcoinNetworkSets.cs]: still spaces here

**Comment by Harumaki-cryptocoin** [comment]
I fixed.

**Pull request merged**